docs/usage.rst

```diff
@@ -128,6 +128,25 @@ Classification
 .. automodule:: tiatoolbox.models.classification.pretrained_info
     :members:
 
+------------
```

Comment:
Is it possible to decouple this PR from segmentation models? To help with the release with patch predictor?

You can add these changes to semantic segmentation PR.